src/test/java/stepDefinitions/SEDTweaksSteps.java

```diff
@@ -12,7 +26,171 @@ public SEDTweaksSteps()
+		assertTrue(driver.findElement(By.cssSelector("div.panel:nth-child(7) > div:nth-child(1) > a:nth-child(1)")).getText().contains("Additional Information"));
+		
+	}
+
+	@Given("^I am looking at a listing's SED details$")
+	public void i_am_looking_at_a_listing_s_SED_details() throws Throwable {
+		driver.get("https://chpl.ahrqstg.org/#/product/8481");
+		WebDriverWait wait = new WebDriverWait(driver, 20);
+		wait.until(ExpectedConditions.presenceOfElementLocated(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)")));
+		
+	}
+
+	@When("^I look at the SED Intended User Description$")
+	public void i_look_at_the_SED_INtended_User_Description() throws Throwable {
+		
+	
+		Thread.sleep(5000);
+		WebElement expanded = driver.findElement(By.cssSelector("div.panel:nth-child(2) > div:nth-child(1) > a:nth-child(1)"));
+		expanded.click();
+					
+		WebElement expanded1 = driver.findElement(By.cssSelector("div.panel:nth-child(4) > div:nth-child(1) > a:nth-child(1)"));
+		expanded1.click();
+		
+		if(driver.getPageSource().contains("No Certification Criteria were tested for SED."))
+		    System.out.println("No Certification Criteria were tested for SED.");
+		else
+			driver.getPageSource().contains("Description of Intended Users");
+	}
+
+	@Then("^the header should be named Description of Intended Users$")
+	public void the_header_should_be_named_Description_of_Intended_Users() throws Throwable {
+	    
+		assertTrue(
+				driver.getPageSource().contains("Description of Intended Users"));
+			
+	}
+
+	@When("^I look at the 'SED Testing Completion Date'$")
+	public void i_look_at_the_SED_Testing_Completion_Date() throws Throwable {
+		if(driver.getPageSource().contains("No Certification Criteria were tested for SED."))
+		    System.out.println("No Certification Criteria were tested for SED.");
+		else
+			driver.getPageSource().contains("Date SED Testing was Completed");
 	
+	}
+
+	@Then("^the header should be named 'Date SED Testing was Completed'$")
+	public void the_header_should_be_named_Date_SED_Testing_was_Completed() throws Throwable {
+	    assertTrue(
+				driver.getPageSource().contains("Date SED Testing was Completed"));
+	}
+
+	@When("^I look at the download button$")
+	public void i_look_at_the_download_button() throws Throwable {
+	   driver.findElement(By.cssSelector(".panel-body > div:nth-child(1) > span:nth-child(1) > span:nth-child(5) > table:nth-child(2) > tfoot:nth-child(2) > tr:nth-child(1) > td:nth-child(1) > button:nth-child(1)")).getTagName();
+	
+	}
+
+	@Then("^the button should be named 'Download SED Details'$")
+	public void the_button_should_be_named_Download_SED_Details() throws Throwable {
+		assertTrue(
+				driver.getPageSource().contains("Download SED Details"));
+	}
+
+	@When("^I look at the associated criteria$")
+	public void i_look_at_the_associated_criteria() throws Throwable {
```

Comment:
This step isn't looking at criteria. It's loading a page. It should be either written to look at criteria or subsumed under the "go to a Listing Details page" step from earlier

Reply:
Agree with moving the Then check to the 'go to a Listing Details page' from earlier.